webui/react/src/services/apiConfig.ts

```diff
@@ -481,7 +481,8 @@ export const getTelemetry: DetApi<EmptyParams, Api.V1GetTelemetryResponse, Telem
 export const getAgents: DetApi<EmptyParams, Api.V1GetAgentsResponse, Type.Agent[]> = {
   name: 'getAgents',
   postProcess: (response) => decoder.jsonToAgents(response.agents || []),
-  request: () => detApi.Cluster.getAgents(),
+  request: () =>
+    detApi.Cluster.getAgents(undefined, undefined, undefined, undefined, undefined, true),
```

Comment:
I have some changes in this branch where I was working toward parameterizing getting slots so that we can still use it for topology.

that branch also includes some changes for disabling topology

basically if and only if  agents < 1000 && slotCount < 10000 then we want to pull down slots and render toplogy